src/Microsoft.Tye.Core/ApplicationFactory.cs

```diff
@@ -59,16 +59,17 @@ public static async Task<ApplicationBuilder> CreateAsync(OutputContext output, F
 
                     project.Build = configService.Build ?? true;
                     project.Args = configService.Args;
+                    foreach (var buildProperty in configService.BuildProperties)
+                    {
+                        project.BuildProperties.Add(buildProperty.Name, buildProperty.Value);
+                    }
                     project.Replicas = configService.Replicas ?? 1;
 
                     await ProjectReader.ReadProjectDetailsAsync(output, project);
 
                     // We don't apply more container defaults here because we might need
                     // to prompt for the registry name.
-                    project.ContainerInfo = new ContainerInfo()
-                    {
-                        UseMultiphaseDockerfile = false,
-                    };
+                    project.ContainerInfo = new ContainerInfo() { UseMultiphaseDockerfile = false, };
```

Comment:
The reformatting changes make it really hard to tell what actually changed. In the future when you send more PRs (and please send more PRs these are great) I'd refrain from doing these changes as they are a bit distracting in the diff.

Reply:
No problem. Your contributing some great stuff! Keep it coming!